src/user/test/unit/user.service.spec.ts

```diff
@@ -58,14 +80,24 @@ describe('UserService', () => {
       .spyOn(userService, 'hashPassword')
       .mockResolvedValue('afuakuasbukaUAASGSA');
 
+    const generateVerificationToken = jest
+      .spyOn(verificationService, 'generateVerificationToken')
+      .mockResolvedValue({ verificationToken: 'testToken' });
+
+    const prismaUpdate = jest
+      .spyOn(prismaService.user, 'update')
+      .mockResolvedValue(mockUpdateResult);
+
     const mailSend = jest.spyOn(mailService, 'sendMail');
 
     const result = await userService.createUser(user);
 
     expect(prismaCreate).toBeCalledTimes(1);
     expect(prismaFindUnique).toBeCalledTimes(1);
     expect(hashedPassword).toBeCalledTimes(1);
+    expect(generateVerificationToken).toBeCalledTimes(1);
     expect(mailSend).toBeCalledTimes(1);
+    expect(prismaUpdate).toBeCalledTimes(1);
```

Comment:
I would not update test cases in that way. 
Your tests should be as compact as possible. Now in one test case, you are asserting a lot of things. 
One test case should cover one part of logic. For example:  you should have a test case that covers if property service method is called to create (prisma.create) the user. 
Another test case to test case to check if mailer method was called and so on.